source/deploying-with-kubernetes/kubernetes-upgrade.rst

```diff
@@ -0,0 +1,212 @@
+    DATA_DIRS[((i++))]="queue/rootcheck"
+    DATA_DIRS[((i++))]="queue/agent-groups"
+    DATA_DIRS[((i++))]="queue/agent-info"
+    DATA_DIRS[((i++))]="queue/agents-timestamp"
+    DATA_DIRS[((i++))]="queue/agentless"
+    DATA_DIRS[((i++))]="queue/cluster"
+    DATA_DIRS[((i++))]="queue/rids"
+    DATA_DIRS[((i++))]="queue/fts"
+    DATA_DIRS[((i++))]="var/multigroups"
+
+Any file that we modify referring to the files previously mentioned, will be changed also the corresponding volume. When the corresponding Wazuh pod is created again, it will get the cited files from the volume, thus keeping the changes made previously.
+
+For a better understanding, we will give an example:
+
+We have our newly created Kubernetes environment following our instructions. In this example, the image of Wazuh used has been ``wazuh/wazuh:3.6.1_6.4.0.``
+
+.. code-block:: bash
+
+    containers:
+    - name: wazuh-manager
+      image: 'wazuh/wazuh:3.6.1_6.4.0'
+
+Let's proceed by creating a set of rules in our `local_rules.xml` file at location `/var/ossec/etc/rules` in our wazuh manager master pod.
+
+.. code-block:: xml
+
+    root@wazuh-manager-master-0:/# vim /var/ossec/etc/rules/local_rules.xml
+    root@wazuh-manager-master-0:/# cat /var/ossec/etc/rules/local_rules.xml
+    <!-- Local rules -->
+
+    <!-- Modify it at your will. -->
+
+    <!-- Example -->
+    <group name="local,syslog,sshd,">
+
+    <!--
+    Dec 10 01:02:02 host sshd[1234]: Failed none for root from 1.1.1.1 port 1066 ssh2
+    -->
+    <rule id="100001" level="5">
+        <if_sid>5716</if_sid>
+        <srcip>1.1.1.1</srcip>
+        <description>sshd: authentication failed from IP 1.1.1.1.</description>
+        <group>authentication_failed,pci_dss_10.2.4,pci_dss_10.2.5,</group>
+    </rule>
+
+    <rule id="100002" level="5">
+        <if_sid>5716</if_sid>
+        <srcip>2.1.1.1</srcip>
+        <description>sshd: authentication failed from IP 2.1.1.1.</description>
+        <group>authentication_failed,pci_dss_10.2.4,pci_dss_10.2.5,</group>
+    </rule>
+
+    <rule id="100003" level="7">
+        <if_sid>5716</if_sid>
+        <srcip>3.1.1.1</srcip>
+        <description>sshd: authentication failed from IP 3.1.1.1.</description>
+        <group>authentication_failed,pci_dss_10.2.4,pci_dss_10.2.5,</group>
+    </rule>
+
+    </group>
+
+This action has modified the `local_rules.xml` file in the `/var/ossec/data/etc/rules` path and in the `/etc/postfix/etc/` rules path due these routes reference our volume assembly points.
+
+.. code-block:: bash
+
+    volumeMounts:
+    - name: config
+      mountPath: /wazuh-config-mount/etc/ossec.conf
+      subPath: ossec.conf
+      readOnly: true
+    - name: wazuh-manager-master
+      mountPath: /var/ossec/data
+    - name: wazuh-manager-master
+      mountPath: /etc/postfix
+
+We can see their content.
+
+.. code-block:: xml
+
+    root@wazuh-manager-master-0:/# cat /var/ossec/data/etc/rules/local_rules.xml
```

Comment:
As @SitoRBJ said, split the command from the output, and format them as console and xml